contracts/factories/PairFactory.sol

```diff
@@ -30,8 +36,38 @@ contract PairFactory is IPairFactory {
         pauser = msg.sender;
         isPaused = false;
         feeManager = msg.sender;
-        stableFee = 2; // 0.02%
-        volatileFee = 2;
+        stableFee = 3; // 0.03%
+        volatileFee = 25; // 0.25%
+        deployer = msg.sender;
+    }
+
+    function setTeam(address _team) external {
+        require(msg.sender == deployer); // might need to set this to deployer?? or just make it
+        require(team == address(0), 'The team has already been set.');
+        team = _team;
+    }
+
+    function setVoter(address _voter) external {
+        require(!initial_voter_set, 'The voter has already been set.');
+        require(msg.sender == deployer); // have to make sure that this can be set to the voter addres during init script
+        voter = _voter;
+        initial_voter_set = true;
+    }
+
+    function setTank(address _tank) external {
+        require(!initial_tank_set, 'The tank has already been set.');
+        require(msg.sender == deployer); // this should be updateable to team but adding deployer so that init script can run..
+        tank = _tank;
+        initial_tank_set = true;
+    }
+
+    function acceptTank(address _tank) external {
+        require(msg.sender == team, 'not pending team');
+        tank = _tank;
+    }
+
+    function getVoter() external view returns (address) {
```

Comment:
this function is pointless, voter is already public

Reply:
Raised this in instruments repo. Removed.